pom.xml

```diff
@@ -294,7 +294,41 @@
         <module>eclipse/ide-target-platform</module>
       </modules>
     </profile>
-
+    <profile>
+      <!-- We only sign the jars when the keystore has been provided in the environment -->
+      <id>sign-jars</id>
+      <activation>
+        <file>
+          <exists>${env.SIGN_KEYSTORE}</exists>
```

Comment:
So, jars are signed only if SIGN_KEYSTORE is an environment variable.
And for CI, this means using github secrets.
Correct?

Reply:
If we do CI for releases yes, we will need a certificate and its password in github secrets. I'm personally not prepared to do that with the official `plugins.gwtproject.org` cert (I make mistakes with code/build wiring, and several CI providers have had breaches), but perhaps we could make a self-signed cert instead for that purpose, just for nightlies?

Reply:
Personally, I think that a self signed is good enough.